### DISCUSSION TITLE
Can you resize to the left, not to the right when you drag to the left, and to resize up when you drag up?

### DISCUSSION THREAD

**Discussion by vap80963** [react-grid-layout/react-grid-layout#1791] [Ideas]
as the title, although we set the top and left resize direction, but we could not resize to that direction truly, we just can resize the component to right or bottom.

**Comment by anreyescl** [comment]
It seems the only properly working behavior is with the standard arrow (bottom right). I added arrows to the left expecting that item will be resized "to the left" but in the end, it does exactly the same as resizing from the right.
